features/support/capybara.rb

```diff
@@ -1,117 +1,48 @@
 # frozen_string_literal: true
 
 require 'selenium/webdriver'
-require 'capybara/minitest'
-include Capybara::Minitest::Assertions
-
-# in case firefox is needed!
-#Capybara.register_driver :selenium do |app|
-#  Capybara::Selenium::Driver.new(app, :browser => :firefox)
-#end
 
 DEFAULT_JS_DRIVER = :headless_chrome
-# in case firefox is needed!
-# DEFAULT_JS_DRIVER = :headless_firefox
-
-Capybara.default_driver = :rack_test
-Capybara.javascript_driver = DEFAULT_JS_DRIVER
-Capybara.default_selector    = :css
-Capybara.disable_animation = true
 
-# Capybara 3 changes the default server to Puma. It can be reverted to the previous default of WEBRick by specifying:
-Capybara.server = :webrick
-
-# see http://www.elabs.se/blog/60-introducing-capybara-2-1
 Capybara.configure do |config|
   config.default_driver = :rack_test
-  config.javascript_driver = DEFAULT_JS_DRIVER
+  config.default_selector = :css
   config.raise_server_errors = true
+
   config.match = :prefer_exact
+  config.javascript_driver = DEFAULT_JS_DRIVER
   config.always_include_port = true
   config.default_max_wait_time = 10
+  config.server = :webrick # default is `:default` (which uses puma)
 end
 
-# Needed because cucumber-rails requires capybara/cucumber
-# https://github.com/cucumber/cucumber-rails/blob/7b47bf1dda3368247bf2d45bcb17a224e80ec6fd/lib/cucumber/rails/capybara.rb#L3
-# https://github.com/teamcapybara/capybara/blob/2.18.0/lib/capybara/cucumber.rb#L17-L19
-Before '@javascript' do
-  Capybara.current_driver = DEFAULT_JS_DRIVER
-end
-
-Before '@chrome' do
-  Capybara.current_driver = :chrome
-end
-
-Before '@firefox' do
-  Capybara.current_driver = :firefox
-end
-
-Around '@security' do |scenario, block|
-  with_forgery_protection(&block)
-end
-
-# monkeypatch to fix
-# not opened for reading (IOError)
-# /cucumber-1.3.20/lib/cucumber/formatter/interceptor.rb:33:in `each'
-# /cucumber-1.3.20/lib/cucumber/formatter/interceptor.rb:33:in `collect'
-# /cucumber-1.3.20/lib/cucumber/formatter/interceptor.rb:33:in `method_missing'
-require 'cucumber/formatter/interceptor'
-class Cucumber::Formatter::Interceptor::Pipe
-  def is_a?(klass)
-    super || klass == IO
-  end
-end
-
-Capybara.register_driver :firefox do |app|
-  Capybara::Selenium::Driver.new(app, browser: :firefox)
-end
```

Comment:
we wont be able to run with firefox?

Reply:
Apparently, not :grimacing: 
Again, I just applied Josemi's suggestion. 

I don't think I have run automated tests in firefox myself, but if you have and want to bring it back, I'll do it.

Reply:
Yeah, I think it is nice to be able to run something with it in case we see strange results with chrome. This is the only browser we ship with RHEL. And Chrome is evil.

Reply:
I agree Chrome is evil, in fact we should use only Firefox, but I guess clients use Chrome. IMO it's fine to keep both.